.ci/terraform/ses.tf

```diff
@@ -0,0 +1,46 @@
+module "ses" {
+  #####################################
+  # Do not modify the following lines #
+  source      = "./module-ses"
+  project     = var.project
+  env         = var.env
+  customer    = var.customer
+  #####################################
+
+  #. aws_region: us-east-1
+  #+ Name of the region where the infrastructure is created
+  aws_region = var.aws_region
+
+  #. vpc_id (required):
+  #+ Amazon VPC id on which create each components.
+  vpc_id                   = "<vpc-id>"
+
+  #. mail_domain (required):
+  #+ Domain used for emailing
+  mail_domain = "example.com"
+
+  #. create_sqs (optional, string): "false"
```

Comment:
could you try with `create_sqs=true` ? If it works well it could help to validate this part too (same for create_elasitcache)